### ISSUE TITLE
Possibly tensorflow related installation issue 

### ISSUE THREAD

**Issue by dominiquesydow** [volkamerlab/TeachOpenCADD#421]
> I also noticed that I couldn't install package teachopencadd where was function seed_everything () because of the tensorflow-base.2.18.0 (something like this). Conda told me that I had corrupted file, pip told me that there was parse-function problem(if I remember correctly). I only want to say that I could resolve this problem with the help of pip install pytorch-lightning=1.7.7, where pytorch-lightning utilities.seed had the same function.

_Originally posted by @Q84-wq in https://github.com/volkamerlab/teachopencadd/issues/419#issuecomment-2266633397_

**Comment by dominiquesydow** [comment]
Hi @AndreaVolkamer, 

Can anyone in your lab please take a look at this? 
If `tensorflow` is a source of problems again, I wonder if we could remove T021 and T022 from our global teachopencadd package environment. 

Why? 
We have had so many issues already with having `tensorflow` as part of the installation and afaiu `tensorflow` is not really the go-to ML/DL package anymore. So why not instead add an installation only for these two talktorials (maybe really with pinned versions).

**Comment by dominiquesydow** [comment]
Hi @Q84-wq,

Could you please add some information on this issue: (a) your OS and (b) how you tried to install `teachopencadd`?

I just tried on MacOS via
`CONDA_SUBDIR=osx-64 mamba create -n teachopencadd teachopencadd`
and was not able to reproduce the error you described. But that might just be because your setup was different.

Many thanks!

**Comment by Q84-wq** [comment]
Hello, miss Sydow, 
I run my laptop on Windows 64-bit (Lenovo) and the app is Jupyter Notebook desktop. I tried again to install teachopencadd. Firstly I installed mamba into conda (I understand that it's not recommended, but it's more convenient for me) https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html, then I used 
mamba create -n teachopencadd teachopencadd -c conda-forge -c defaults
and there was written "warning libmamba Problem type not implement SOLVER_RULE_STRICT_REPO_PRIORITY" many times, 
and then "Could not solve the environment specs
The following packages are incompatible" and there were biopython - > python, numpy; biotite - > python, h5py 2.10.0 - > python_abi 3.6(3.7).* *_cp36(37)m, mdanalysis - > python_abi; tensorflow 1.14.0, numpy - > python
The second way, mamba env create -f https://raw.githubusercontent.com/volkamerlab/TeachOpenCADD/master/devtools/test_env.yml
conda activate teachopencadd, it has stuck on Solving environment for a really long time
Thank you very much!

**Comment by mbackenkoehler** [comment]
The global environment is now replaced by local per-talktorial environments.
